### DISCUSSION TITLE
How to import database with SQL .txt script?

### DISCUSSION THREAD

**Discussion by alinakrav** [supabase/supabase#1622] [Questions] [answered]
I'm new to database hosting, and I have a script that describes tables and their columns in a .txt file. How can I import this database into my supabase project?

**Comment by steve-chavez** [comment, marked as answer]
Hey @alinakrav,

What's the format of the txt file? Is it [CSV(comma-separated values)](https://en.wikipedia.org/wiki/Comma-separated_values#Example)?

If so, you could import them directly within the UI, on the Table Editor - click table icon -> + New table -> Add existing content to new table -> Upload CSV.
